lib/cube_browser/__init__.py

```diff
@@ -0,0 +1,48 @@
+import matplotlib.pyplot as plt
+
+
+class Contourf(object):
+    def __init__(self, cube, coords, **kwargs):
+        self.cube = cube
+        self.coords = coords # coords to plot
+        self.kwargs = kwargs
+        self.axes = self.new_axes()
+        self.qcs = None  # QuadContourSet
+        # A mapping of 1d-coord name to dimension
+        self.coord_dim = self.coord_dims()
+
+    # This constructs the original and updated plots
+    def __call__(self, **kwargs):
+        index = [slice(None)] * self.cube.ndim
+        for name, value in kwargs.items():
+            if name in self.coord_dim:
+                index[self.coord_dim[name]] = value
+        cube = self.cube[tuple(index)]
+        plt.sca(self.axes)
+        self.qcs = iplt.contourf(cube, coords=self.coords,
+                                 axes=self.axes, **self.kwargs)
+        return plt.gca()
+
+    def new_axes(self):
```

Comment:
a comment here that this is temporary (if it is) would be helpful
have we made an issue for managing axes yet?